src/python/WMCore/Services/WMArchive/DataMap.py

```diff
@@ -417,9 +417,13 @@ def createArchiverDoc(job, version=None):
         import WMCore
         version = WMCore.__version__
     # append meta data in fwjr
+    wnName = ""
+    if "WorkerNodeInfo" in fwjr:
+        wnName = fwjr["WorkerNodeInfo"].get("HostName", "")
+
     newfwjr['meta_data'] = {'agent_ver': version,
                          'host': socket.gethostname().lower(),
-                         'wn_info': fwjr["WorkerNodeInfo"],
+                         'wn_name': wnName,
```

Comment:
@amaltaro, we talk about WorkerNodeName as key but it seems everything else is lower case for meta_data. So I change to wn_name, hope that is OK.